### PR TITLE
i18n(pt-BR): Add `layouts.md` and `framework.components.md` translations

### PR DESCRIPTION
**Pull request by Yan-Thomas**
Translations for both `layouts.md` and `framework.components.md` pages.

### CODE REVIEW

File: src/pages/pt-BR/core-concepts/framework-components.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import vue from '@astrojs/vue';
+import solid from '@astrojs/solid-js';
+import lit from '@astrojs/lit';
+
+export default defineConfig({
+	integrations: [react(), preact(),svelte(), vue(), solid() , lit()],
+});
+```
+
+⚙️ Veja o [Guia de Integrações](/pt-BR/guides/integrations-guide) para mais detalhas em como instalar e configurar integrações Astro.
```
Comment (caioferrarezi) [inline src/pages/pt-BR/core-concepts/framework-components.md:38]:
```suggestion
⚙️ Veja o [Guia de Integrações](/pt-BR/guides/integrations-guide) para mais detalhes em como instalar e configurar integrações Astro.
```

File: src/pages/pt-BR/core-concepts/framework-components.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+<ContagemInterativa client:visible />
+```
+
+> ⚠️ Qualquer JS de renderização necessário para o componente de framework (e.x. React, Svelte) é baixado com a página. As diretivas `client:*` apenas ditam quando o _JS do componente_ é importado e quando o _componente_ é hidratado. 
+
+### Diretivas de Hidratação Disponível
+
+Há diversas diretivas de hidratação disponíveis para componentes de frameworks de UI: `client:load`, `client:idle`, `client:visible`, `client:media={QUERY}` e `client:only=" "`
+
+📚 Veja nossa página de [referência de diretivas](/pt-BR/reference/directives-reference#client-directives) para uma completa descrição destas diretivas de hidratação e seus usos.
```
Comment (caioferrarezi) [inline src/pages/pt-BR/core-concepts/framework-components.md:91]:
```suggestion
📚 Veja nossa página de [referência de diretivas](/pt-BR/reference/directives-reference#client-directives) para uma descrição completa destas diretivas de hidratação e seus usos.
```

**Comment by caioferrarezi** [comment]
Great work, @Yan-Thomas!

**Comment by Yan-Thomas** [comment]
Thanks for the review @caioferrarezi!! 🚀

**Review by caioferrarezi — APPROVED** [review]
LGTM 🚀

**Pull request merged**